x-pack/plugins/cloud_security_posture/public/components/no_findings_states.tsx

```diff
@@ -27,6 +27,10 @@ import type { IndexDetails } from '../../common/types';
 
 const REFETCH_INTERVAL_MS = 20000;
 
+interface PostureTypes {
+  posturetype: 'cspm' | 'kspm';
```

Comment:
Since we already of `'cspm' | 'kspm'` define as a type in  `CloudSecurityPolicyTemplate`. Can we reuse the  ```
`CloudSecurityPolicyTemplate` type?  For example, something like
```
 interface PostureTypes {
  postureType: CloudSecurityPolicyTemplate;
}
```

```
 I  wonder if we should change the name `PostureTypes` to `PolicyTemplates` because the response data object will contain perhaps three types of policy templates `cspm`, `kspm` , and `vul-mgmt` ?
 
```
 interface PolicyTemplates {
    policyTemplate: CloudSecurityPolicyTemplate
 }
```